app/src/main/java/mozilla/lockbox/view/OnboardingFingerprintAuthFragment.kt

```diff
@@ -0,0 +1,88 @@
+package mozilla.lockbox.view
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.jakewharton.rxbinding2.view.clicks
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.subjects.PublishSubject
+import kotlinx.android.synthetic.main.onboarding_biometric_unlock.view.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import mozilla.lockbox.R
+import mozilla.lockbox.presenter.OnboardingFingerprintAuthPresenter
+import mozilla.lockbox.presenter.OnboardingFingerprintView
+import mozilla.lockbox.model.FingerprintAuthCallback
+import mozilla.lockbox.support.Constant
+import mozilla.lockbox.view.OnboardingFingerprintAuthFragment.AuthCallback.OnAuth
+import mozilla.lockbox.view.OnboardingFingerprintAuthFragment.AuthCallback.OnError
+
+@ExperimentalCoroutinesApi
+class OnboardingFingerprintAuthFragment : Fragment(), OnboardingFingerprintView {
+    private val compositeDisposable = CompositeDisposable()
+
+    private val _authCallback = PublishSubject.create<AuthCallback>()
+    override val authCallback: Observable<AuthCallback> get() = _authCallback
+
+    override val onDismiss: Observable<Unit>
+        get() = view!!.skipOnboarding.clicks()
+
+    private var isEnablingDismissed: Boolean = true
+
+    sealed class AuthCallback : FingerprintAuthCallback {
```

Comment:
why did we keep an `AuthCallback` around, but rename the root class to `FingerprintAuthCallback`? This seems like some unnecessary shuffling to me.